example/grammar/magnet.cpp

```diff
@@ -0,0 +1,496 @@
+//
+// Copyright (c) 2022 alandefreitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0.
+// https://www.boost.org/LICENSE_1_0.txt
+//
+
+#include <boost/url/url_view.hpp>
+#include <boost/url/rfc/absolute_uri_rule.hpp>
+#include <boost/url/grammar/charset.hpp>
+#include <iostream>
+
+using namespace boost::urls;
```

Comment:
hmm idk about this, we might need a namespace alias and `url::` or `urls::` in front of everything

Reply:
I thought https://github.com/CPPAlliance/url/issues/241 applied here